### PR TITLE
Add workflow for manual image build

### PR DESCRIPTION
**Pull request by bensteinberg**
https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Build images
+
+on: [workflow_dispatch]
```
Comment (kilbergr) [inline .github/workflows/build.yml]:
```suggestion
on: workflow_dispatch
```

Unless you intend to add other actions, you can take this out of a list as demonstrated here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

**Review by kilbergr — APPROVED** [review]
This looks reasonable to me. I had one comment around syntax.

Also, reviewing the tests workflow, I noticed you had the following conditional:
`if: github.event_name == 'push' && github.ref == 'refs/heads/develop'`
for pushing the docker image. Do you want to include that here?

**Comment by bensteinberg** [comment]
> Also, reviewing the tests workflow, I noticed you had the following conditional: `if: github.event_name == 'push' && github.ref == 'refs/heads/develop'` for pushing the docker image. Do you want to include that here?

I think not -- this workflow should never happen automatically, so `github.event_name` will never be `push`, and workflows triggered by `workflow_dispatch` must always be in the default branch, so I think `github.ref` will always point to develop.

**Comment by kilbergr** [comment]
Ah! Thank you for the explanation!

**Pull request merged**